<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kitaplar</class>
 <widget class="QDialog" name="kitaplar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_kitaplar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>321</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_oduncAlinma">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>320</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_oncedenOdunAlinma">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>510</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kitapNo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kitapAd">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_stok">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="byn_yeni_kayit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Yeni kayıt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guncelle">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>güncelle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sil">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>sil</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tüm Kitaplar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>KİTAP İŞLEMLERİ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kitap no</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kitap Ad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>seçilen kitabın ödünç alınma durumu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>seçilen kitabın daha önceden ödünç alınma durumu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
